src/providers/Theme/README.md

```diff
@@ -0,0 +1,177 @@
+# Theme
```

Comment:
Awesome! :D

Just please check it out.
- It should be in other PR for `hackatalk-mobile` to merge `master`.
- This `README.md` is for every component, not only for [Slider].